virttest/libvirt_xml/devices/interface.py

```diff
@@ -232,3 +251,53 @@ def __init__(self, virsh_instance=base.base.virsh):
                                      tag_name="guest")
             super(self.__class__, self).__init__(virsh_instance=virsh_instance)
             self.xml = '<driver/>'
+
+    class Vlan(base.base.LibvirtXMLBase):
+
+        """
+        Interface vlan xml class.
+
+        Properties:
+
+        vlan:
+            attribute.
+        tags:
+            list. tags element dict list
+         """
+        __slots__ = ("vlan_trunk", "tags")
```

Comment:
@yalzhang Is there a reason why you defined `vlan_trunk` instead of `trunk`? During my manual test I intuitively used `iface.vlan['trunk'] = 'yes'` instead of `iface.vlan['vlan_trunk'] = 'yes'` leading to a KeyError. IMO if there's no specific reason to call it otherwise, it should be better `trunk`.

Reply:
@yalzhang even I feel naming the slot identifier as just `trunk` would be for better usability, if there are no functional reasons to be `vlan_trunk`, otherwise LGTM.

Reply:
Yes, you are right, updated. Thank you!